src/v2i-hub/ERVCloudForwardingPlugin/src/ERVCloudForwardingPlugin.h

```diff
@@ -50,7 +50,11 @@ namespace ERVCloudForwardingPlugin
         string _rsuName;
         uint16_t _snmpPort;
         // V2X Hub sends RSU register request to carma-cloud at specified interval. Default 3600 seconds
-        uint32_t _rsuInterval=3600;
+        uint32_t _rsuInterval = 3600;
+        // Max attempts to try to make connection to RSU and send the RSU location to the cloud.
+        uint32_t _max_rsu_connection_attempt = 30;
+        // Waiting to start the web service
+        uint32_t _max_web_service_waiting = 10; // unit of second
```

Comment:
The configuration parameter does not work for this case. I tested this earlier that it can only read the initial value and the service in the plugin cannot read the updated parameters.

Reply:
Does that mean that if the configuration is updated on V2XHub for this plugin, the plugin will not be updated (but for other plugins the configuration can be updated dynamically)? If so, if it would take some time to investigate and fix, could we at least create a GitHub issue for this?